File: src/main/java/com/train/Tester.java
package com.train;

import java.sql.SQLOutput;
import java.util.Scanner;

public class Tester {
    public static void main(String[] args) {
        Ticket singleTrip = new Ticket("台北","高雄", 1000);
        Ticket roundTrip = new RoundTrip("台北","高雄", 2000);
        Scanner input = new Scanner(System.in);
        System.out.println("Please enter number of tickets:");
        int numberOfTicket = input.nextInt();
        System.out.println("How many round-trip tickets:");
        int numberOfRoundTicket = input.nextInt();
        System.out.println("Total Tickets: " + numberOfTicket);
        System.out.println("Round-trip: " + numberOfRoundTicket);
        System.out.println("Total: " + (singleTrip.calc(numberOfTicket-numberOfRoundTicket) + roundTrip.calc(numberOfRoundTicket)));
    }
}

File: src/main/kotlin/com/train/Tester.kt
package com.train

import java.util.*

fun main(args: Array<String>) {
    val singleTrip = Ticket("台北", "高雄", 1000)
    val roundTrip = RoundTrip("台北", "高雄", 2000)
    val input = Scanner(System.`in`)
    println("Please enter number of tickets:")
    val numberOfTicket = input.nextInt()
    println("How many round-trip tickets:")
    val numberOfRoundTicket = input.nextInt()
    println("Total Tickets: " + numberOfTicket)
    println("Round-trip: " + numberOfRoundTicket)
    println("Total: " + (singleTrip.calc(numberOfTicket - numberOfRoundTicket) + roundTrip.calc(numberOfRoundTicket)))
}
